README.md

```diff
@@ -1 +1,44 @@
-# space-travelers-hub2.0
\ No newline at end of file
+# Space-Travelers-Hub
+- In this task, we will be working with the real live data from the SpaceX API. Our task is to build a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+# Screenshots
+## Rocket and Home Page:
+## Missions Page:
+## Dragons Page:
```

Comment:
- [ ] Let's include screenshots of this great project in the README so that visitors can see the page previews and to also make the README more professional.